src/open_dread_rando/files/randomizer_powerup.lua

```diff
@@ -262,7 +262,9 @@ local tItemTunableHandlers = {
     end,
     ["ITEM_UPGRADE_SPEED_BOOST_CHARGE"] = function(quantity)
         -- Amount of time in seconds for SB to charge - vanilla is 1.5 seconds. Each upgrade reduces by 0.25 seconds.
-        local chargeTime = math.max(0.25, 1.5 - quantity * 0.25)
+        -- Cannot be <= 0 or else all hell breaks loose.
+        -- SB activation is very buggy <= 0.5, so we clamp it a tiny bit higher.
+        local chargeTime = math.max(0.55, 1.5 - quantity * 0.25)
```

Comment:
would 0.51 work?

Reply:
I haven't tried it, but that would only be one extra frame beyond when it has a problem (technically it'd be effectively rounded up to 0.5167). 0.55 is 3-4 frames extra.